### PR TITLE
Update for 8.0b0

### PR DESCRIPTION
**Pull request by hjoliver**
Combining extant PRs into one...

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 974480e2cee98071470ce8ca13eab600a32f213310b9fdc14a0b15ab6d50cb07
```
Comment (kinow) [inline recipe/meta.yaml:10]: This probably need changing too!

**Review by kinow — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (hjoliver) [reply recipe/meta.yaml:10]: Oops, yeah. I'm a newbie at this conda feedstock stuff!

Comment (hjoliver) [reply recipe/meta.yaml]: Hash updated.

**Review by kinow — APPROVED** [review]
Looks good! Downloaded the tar.gz to verify the hash :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,17 +19,19 @@ requirements:
...
+    - aiofiles >=0.5.0,<0.6.0
     - ansimarkup >=1.0.0
     - colorama >=0.4,<0.5
     - click >=7.0
     - cryptography
     - graphene >=2.1,<3
     - jinja2 ==2.11.0,<2.12
     - metomi-isodatetime ==1!2.0.1
-    - protobuf ==3.12.1
+    - protobuf =>3.15.0,<3.16.0
```
Comment (hjoliver) [inline recipe/meta.yaml:30]:
```suggestion
    - protobuf >=3.15.0,<3.16.0
```

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**